profile/arc_cwl_ro_crate.md

```diff
@@ -23,11 +23,14 @@ To continue staying consisten with the [ISA RO-Crate Profile](https://github.com
 
 ```mermaid
 flowchart TD
-        A["File\nSoftwareSourceCode\nComputationalWorkflow"] -- "input\noutput" --> B["FormalParameter"]
-        A -- "instrument" --> C["CreateAction"]
+        A["File\nSoftwareSourceCode\nComputationalWorkflow\nLabProtocol"] -- "input\noutput" --> B["FormalParameter"]
+        A -- "instrument" --> C["CreateAction\nLabProcess"]
+        C -- "executes" --> A
         C -- "agent" --> D["Person or Organization"]
         B -- "exampleOfWork" --> E["File or Property Value"]
         C -- "object result" --> E
+        D["Assay=Dataset"] -- "processSequence" --> C
```

Comment:
At the moment, the property `processSequence` of an Assay has not been added to the bioschemas specification and has been mapped to `about`.